server/src/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -404,7 +416,10 @@ private boolean validateVolume(Account caller, long ownerId, Long zoneId, String
             }
             UriUtils.validateUrl(format, url);
         // check URL existence
```

Comment:
@yvsubhash can you fix the comment indentation as well, thanks

Reply:
@rhtyd  its taken care.